site/content/en/latest/tasks/traffic/gateway-address.md

```diff
@@ -21,21 +22,22 @@ The Envoy Gateway deploys Envoy Proxy Service as `LoadBalancer` by default,
 so you can set the address of the Gateway directly (the address settings here are for reference only):
 
 ```shell
-kubectl patch gateway eg --type=json --patch '[{
-   "op": "add",
-   "path": "/spec/addresses",
-   "value": [{
-      "type": "IPAddress",
-      "value": "1.2.3.4"
-   }]
-}]'
+kubectl patch gateway eg --type=json --patch '
+- op: add
+  path: /spec/addresses
+  value:
+   - type: IPAddress
+     value: 1.2.3.4
+'
 ```
 
 Verify the Gateway status:
 
 ```shell
 kubectl get gateway
+```
 
+```console
```

Comment:
imo, we can use markdown style.  console doesn't seem to highlight any code in md.

```markdown
NAME   CLASS   ADDRESS   PROGRAMMED   AGE
eg     eg      1.2.3.4   True         14m
```

```console
NAME   CLASS   ADDRESS   PROGRAMMED   AGE
eg     eg      1.2.3.4   True         14m
```